static/404.html

```diff
@@ -0,0 +1,29 @@
+<!DOCTYPE html>
+<html lang='en'>
+<head>
+    <title>
+        404 Error | Sauce Docs
+    </title>
+    <link rel="stylesheet" href="/css/404.css">
+    <link data-react-helmet="true" rel="shortcut icon" href="/img/favicon.ico">
+</head>
+<body>
+    <div class="container">
+        <div class="flex header"><h2>404 Error | Sauce Docs: URL does not exist</h2></div>
+        <div class="flex sauce_bot">
+            <img class="gif" src="/img/Manual_Bot_EXPORT.gif" alt="Sauce Bot Animated GIF">
+        </div>
+        <div class="flex message">
+                <p>Oops, it looks like the page you're looking for either no longer exists or moved.</p>
```

Comment:
@spider-sauce sorry, one more thing! Would you be ok with change? 
_Oops! It looks like the page you're looking for no longer exists or has been moved._